src/amazon-sqs/provision

```diff
@@ -37,7 +37,7 @@ if [[ -e "${INPUT_DIR}/tags" ]]; then
 fi
 
 # Choose a queue name that's unique to this resource in this sandbox.
-QUEUE_NAME="signadot-${SANDBOX_ID}-${RESOURCE_NAME}"
+QUEUE_NAME="signadot-${RESOURCE_NAME,,}-${SANDBOX_ID}"
```

Comment:
seems more uniform lowercased to me.  In any event, the ordering is more readable with resource name first.

Reply:
I agree that it's nicer to put the human-readable name before the random ID.

If there isn't a technical reason to lowercase in given plugin, though, I'd rather let the user's input pass through. We allowed capitals in the resource name to give them the choice of writing names in a form that they feel is more readable. I'd like to preserve the user's choice whenever it's feasible.

Reply:
I was thinking of the user in terms of those interacting with the cluster, where lower casing is more uniform so passing through the casing would make workflows there, such as force delete,  more complex because of the lack of identity correspondence between the resource CRD name and the queue name.  Anyway, it's a tradeoff.